api/app/signals/apps/signals/managers.py

```diff
@@ -603,3 +615,24 @@ def update_type(self, data, signal):
                                                                   type=signal_type, prev_type=previous_type))
 
         return signal_type
+
+    def _update_directive_departments_no_transaction(self, data, signal):
+        from signals.apps.signals.models.directing_departments import DirectingDepartments
+
+        directing_departments = DirectingDepartments.objects.create(_signal=signal, created_by=data['created_by'])
+        for department_data in data['departments']:
+            directing_departments.departments.add(department_data['id'])
+
+        signal.directing_departments_assignment = directing_departments
+        signal.save()
+
+        return directing_departments
+
+    def update_directive_departments(self, data, signal):
```

Comment:
"update_directive_departments" -> "update_directing_departments" to match rest of this PR.

Reply:
Forgot to change this to match the rest of the feature, oops 😂